build_tools/benchmarks/run_benchmarks_on_android.py

```diff
@@ -91,7 +90,7 @@ def adb_push_to_tmp_dir(
 def adb_execute_and_get_output(
     cmd_args: Sequence[str],
     relative_dir: pathlib.PurePosixPath = pathlib.PurePosixPath(),
-    verbose: bool = False) -> str:
+    verbose: bool = False) -> Tuple[str, str]:
```

Comment:
I think the pydoc below also needs to update.

Reply:
The pydoc says that it returns both stdout and stderr? Not sure I follow!

Reply:
Ah I was looking at the wrong file – thanks!